### PR TITLE
Fix extra error print in failed load

### PR DESCRIPTION
**Pull request by bolasim**

### CODE REVIEW

**Review by squidarth — APPROVED** [review]

File: truss/templates/control/control/endpoints.py [hunk 1]
```diff
@@ -59,7 +60,7 @@ async def proxy(request: Request):
 
                 if await _is_model_not_ready(resp):
                     raise ModelNotReady("Model has started running, but not ready yet.")
-            except ConnectionError as exp:
+            except (ConnectionError) as exp:
```
Comment (squidarth) [inline truss/templates/control/control/endpoints.py:63]: actually sorry we should replace ConnectionError here w/ ConnectError, RemoteProtocolError

Comment (bolasim) [reply truss/templates/control/control/endpoints.py:63]: I tried that and didn't behave as expected locally. let me try again

**Review by squidarth — APPROVED** [review]
